tests/e2e/runtime/runtime_test.go

```diff
@@ -49,16 +47,9 @@ func getAPIResponse(t *testing.T, testName, runtimeExternalURL string) (*daprAPI
 	// this is the publish app's endpoint, not a dapr endpoint
 	url := fmt.Sprintf("http://%s/tests/%s", runtimeExternalURL, testName)
 
-	resp, err := http.Get(url)
-	defer resp.Body.Close()
-
+	body, err := utils.HTTPGet(url)
 	require.NoError(t, err)
-	require.Equal(t, resp.StatusCode, http.StatusOK)
```

Comment:
do not remove the status code check.

Reply:
The return value of `HTTPGet` isn't http.Response any more but just byte[]

Reply:
Switch to `HTTPGetRaw` to keep the same logic. I only moved the `nil` deref down.